content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst

```diff
@@ -9,122 +9,161 @@ value.
 .. image:: using_inventory_valuation/add-manual-valuation.png
    :align: center
    :alt: Add manual valuation of stock value to a product.
 
 Create manual valuation entries on the :guilabel:`Stock Valuation` dashboard by first navigating to
-:menuselection:`Inventory app --> Reporting --> Inventory Valuation`, and then click the little gray
-drop-down icon to the left of a product's name, to then reveal stock valuation line items below as
-well as a teal :guilabel:`+ (plus)` button on the right.
+:menuselection:`Inventory app --> Reporting --> Valuation`. Next, to enable the *product
+revaluation* feature, select :menuselection:`Group by --> Product` to organize all the records by
+product. Click on the gray :guilabel:`▼ (drop-down)` icon to reveal stock valuation line items
+below, as well as a teal :guilabel:`+ (plus)` button on the right.
 
-Click the teal :guilabel:`+ (plus)` button to open up the :guilabel:`Product Revaluation` form,
-where updates to the calculation of inventory valuation can be made, by increasing or decreasing the
+Click the teal :guilabel:`+ (plus)` button to open up the :guilabel:`Product Revaluation` form.
+Here, the inventory valuation for a product can be recalculated, by increasing or decreasing the
 unit price of each product.
 
 .. image:: using_inventory_valuation/product-revaluation.png
    :align: center
    :alt: Product revaluation form adding a value of $1.00 with the reason being inflation.
+
+.. note::
+   This button is only visible after grouping entries by product.
+
+Inventory valuation journal entries
+-----------------------------------
+
+In Odoo, automatic inventory valuation records are also recorded in the :menuselection:`Accounting
+app --> Accounting --> Journal Entries`. On this comprehensive list of accounting entries, inventory
+valuation entries are identified by checking values in the :guilabel:`Journal` column, or looking
+for the :guilabel:`Reference` column value which matches the warehouse operation reference (e.g.
+`WH/IN/00014` for receipts).
+
+Clicking on an inventory valuation journal entry opens a *double-entry accounting* record. These
+records are generated by Odoo to track the change of value in inventory valuation as products are
+moved in and out of the warehouse.
+
+.. example::
+   To view the inventory valuation of 10 *tables*, costing $10.00 each, upon reception from the
+   vendor, go to the :menuselection:`Journal Entries` page found in :menuselection:`Accounting app
+   --> Accounting --> Journal Entries`. Here, click the journal line where the :guilabel:`Reference`
+   column value matches the reference on the receipt, `WH/IN/00014`.
+
+   .. image:: using_inventory_valuation/stock-valuation-product.png
+      :align: center
+      :alt: Stock valuation page depicting the products within a shipment.
+
+   *Stock interim* is a holding account for money intended to pay vendors for the product. The
+   *stock valuation* account stores the value of all on-hand stock.
```

Comment:
If we're pointing to / describing UI elements, then these get guis

Reply:
This one is tough. These are journal accounts that Odoo creates by default, but they can be changed.
![image](https://github.com/odoo/documentation/assets/23611711/7b96dc6e-c47f-4727-9972-2766847a9d70)
What do you think, Zac? `stock interim` or *stock interim* ?

Reply:
Would do backticks `stock interim` since it's a field value 👍